bn_mp_to_radix.c

```diff
@@ -43,11 +53,12 @@ mp_err mp_to_radix(const mp_int *a, char *str, size_t maxlen, int radix)
       /* subtract a char */
       --maxlen;
    }
-
-   digs = 0;
+   digs = 0u;
    while (!MP_IS_ZERO(&t)) {
       if (--maxlen < 1u) {
          /* no more room */
+         /* TODO: It could mimic mp_to_radix_n if that is not an error
+                  or at least not this error (MP_ITER or a new one?). */
```

Comment:
maybe add an `MP_BOF`? But we should add that in another PR as this will be used in more functions than this one!

Reply:
Good idea! But I would prefer MP_OVF or sth else. BOF reminds me of EOF.

Reply:
well a regular overflow shouldn't be possible in an MPI library *rimshot* 
... and `MP_BUFFER_OVERFLOW` is too long ... `MP_BOVF`?

Reply:
Ok or MP_BUF?

Reply:
I thought about that too... fine by me.

Reply:
> BOF reminds me of EOF but that doesn't make sense
BOF reminds me more of BOFH ;-)

So `MP_BUF` it is?

> we should add that in another PR

There isn't that much else I think, but as it doesn't cost anything: new PR.

Reply:
See #357